applications/nrf5340_audio/doc/building.rst

```diff
@@ -42,19 +42,27 @@ You can now program the development kit.
 Building and programming using script
 *************************************
 
-The suggested method for building each of the applications and programming it to the development kit is running the :file:`buildprog.py` Python script, which is located in the :file:`applications/nrf5340_audio/tools/buildprog` directory.
+The suggested method for building each of the applications and programming it to the development kit is running the :file:`buildprog.py` Python script.
 The script automates the process of selecting :ref:`configuration files <nrf53_audio_app_configuration_files>` and building different applications.
 This eases the process of building and programming images for multiple development kits.
 
+The script is located in the :file:`applications/nrf5340_audio/tools/buildprog` directory.
+
 Preparing the JSON file
 =======================
 
 The script depends on the settings defined in the :file:`nrf5340_audio_dk_devices.json` file.
-Before using the script, make sure to update this file with the following information for each development kit you want to use:
+Before using the script, make sure to update this file with the following information for each development kit you want to use.
+This is how the file looks by default:
+
+.. literalinclude:: ../tools/buildprog/nrf5340_audio_dk_devices.json
+    :language: json
+
+When preparing the JSON file, update the following fields:
 
 * ``nrf5340_audio_dk_snr`` -- This field lists the SEGGER serial number.
-  You can check this number on the sticker on the nRF5340 Audio development kit.
-  Alternatively, connect the development kit to your PC and run ``nrfjprog -i`` in a command window to print the SEGGER serial number of the kit.
+  You can check this five-digit number on the sticker on the nRF5340 Audio development kit.
```

Comment:
The json-file needs the entire 10-digit number to flash. There is a 12-digit number when running nrfutil device list, so if the first 2 zeros are removed we have the correct 10-digit number